Runtime error

Go v1.11.4
macOS 10.14.2

Hello, I've just installed using `go get github.com/hSATAC/wifi-password-qr` and then ran `wifi-password-qr`. This is what I got...

```
wifi-password-qr
panic: runtime error: index out of range

goroutine 1 [running]:
main.getWifiInfoFromRegex(0x0, 0x0, 0x412c852, 0xf, 0x0, 0x0)
	/Users/carlca/Code/go/src/github.com/hSATAC/wifi-password-qr/main.go:73 +0x85
main.getWifiInfo(0x41f5488, 0x41f5488, 0x41f5480, 0x41f5480)
	/Users/carlca/Code/go/src/github.com/hSATAC/wifi-password-qr/main.go:57 +0xe3
main.main()
	/Users/carlca/Code/go/src/github.com/hSATAC/wifi-password-qr/main.go:17 +0x34
```

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module wifi-password-qr
 
 require (
-	github.com/keybase/go-keychain v0.0.0-20181011010623-f1daa725cce4 // indirect
-	github.com/mdp/qrterminal v1.0.1 // indirect
+	github.com/keybase/go-keychain v0.0.0-20181011010623-f1daa725cce4
+	github.com/mdp/qrterminal v1.0.1
 )

--- a/main.go
+++ b/main.go
@@ -2,12 +2,13 @@ package main
 
 import (
 	"fmt"
-	"github.com/keybase/go-keychain"
-	"github.com/mdp/qrterminal"
 	"os"
 	"os/exec"
 	"regexp"
 	"strings"
+
+	"github.com/keybase/go-keychain"
+	"github.com/mdp/qrterminal"
 )
 
 const airportBin = "/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport"
@@ -24,7 +25,12 @@ func main() {
 	fmt.Printf("\033[90m … getting password for \"%s\". \033[39m\n", ssid)
 	fmt.Println("\033[90m … keychain prompt incoming. \033[39m")
 
-	password := getWifiPassword(ssid)
+	password, err := getWifiPassword(ssid)
+
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Could not retrieve password. %v", err)
+		os.Exit(1)
+	}
 
 	if password == "" {
 		fmt.Fprintln(os.Stderr, "Could not retrieve password.")
@@ -38,17 +44,23 @@ func main() {
 	displayQRCode(qr)
 }
 
-func getWifiPassword(ssid string) (password string) {
+func getWifiPassword(ssid string) (password string, err error) {
 	query := keychain.NewItem()
 	query.SetSecClass(keychain.SecClassGenericPassword)
 	query.SetDescription("AirPort network password")
 	query.SetAccount(ssid)
 	query.SetMatchLimit(keychain.MatchLimitOne)
 	query.SetReturnData(true)
-	results, _ := keychain.QueryItem(query)
-	password = string(results[0].Data)
+	results, err := keychain.QueryItem(query)
+	if err != nil {
+		return "", err
+	} else if len(results) != 1 {
+		return "", nil
+	} else {
+		password = string(results[0].Data)
+	}
 
-	return password
+	return password, err
 }
 
 func getWifiInfo() (authenticationType string, ssid string) {

